python-checks/src/test/java/org/sonar/python/checks/MandatoryFunctionReturnTypeHintCheckTest.java

```diff
@@ -20,11 +20,170 @@
 package org.sonar.python.checks;
 
 import org.junit.Test;
+import org.sonar.python.checks.quickfix.PythonQuickFixVerifier;
 import org.sonar.python.checks.utils.PythonCheckVerifier;
 
+import static org.sonar.python.checks.utils.CodeTestUtils.code;
+
 public class MandatoryFunctionReturnTypeHintCheckTest {
   @Test
   public void test() {
     PythonCheckVerifier.verify("src/test/resources/checks/mandatoryFunctionReturnType.py", new MandatoryFunctionReturnTypeHintCheck());
   }
+
+  @Test
+  public void testQuickFixAddReturnTypeLiteral() {
```

Comment:
_minor_ The usual :)
```suggestion
  public void quick_fix_add_return_type_literal() {
```

Reply:
Sadly, it is not as easy with the inferred type to detect if it could be a generic type. I think I will have to approach this by only taking care of nongeneric built-in types.